src/api.js

```diff
@@ -255,90 +255,6 @@
     "FAILED"
   ]);
 
-  /*----------------------------------------------------------------
```

Comment:
Is the removal of those lines expected?

Reply:
Yes, these are duplicated enums. We have the same enums above these